### PR TITLE
Added more params to covidconnect api

### PR DESCRIPTION
**Pull request by minkesh**
Integrated the covidconnect api with the new set of parameters mentioned in this PR https://github.com/viksit/covidconnect/pull/14

### CODE REVIEW

**Comment by bodhish** [comment]
@viksit can you review this?

File: components/DetailedHome.jsx [hunk 1]
```diff
@@ -184,7 +184,7 @@ export default function DetailedHome({ state, district, type }) {
                                 />
                            )}
                             {tabVal === 'twitter' && (
-                                <TwitterContainer searchStr={mapDistrictToCity(districtChoosen)} />
+                                <TwitterContainer searchStr={mapDistrictToCity(districtChoosen)} resourceType={resourceChoosen} />
```
Comment (viksit) [inline components/DetailedHome.jsx:187]: minor: typo fix "choosen" to "chosen"

File: components/Selector.jsx [hunk 2]
```diff
@@ -9,45 +9,12 @@ import { faTwitter } from '@fortawesome/free-brands-svg-icons';
...
-    return [data, loading];
-}
-export { useFetch };
+import useFetch from '@hooks/use-fetch';
 
 const Selector = ({ data, page, placeholder, localeState, localeDistrict }) => {
     const [searchStr, setSearchStr] = useState('');
     const [editing, setEditing] = useState(false);
-    const [covidConnectResults, loading] = useFetch(searchStr);
+    const [covidConnectResults, loading] = useFetch({ city: searchStr });
```
Comment (viksit) [inline components/Selector.jsx:17]: so the home page shows all materials, and the individual page will show based on the dropdown?

File: hooks/use-fetch.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+
+const BASE_URL = 'https://covidconnect.vercel.app/api/1.1/data';
+
+function useFetch({ city, materialType, resourceType = 'supply', maxResults = 25 }) {
+
+  const url = new URL(BASE_URL);
```
Comment (viksit) [inline hooks/use-fetch.js:7]: TIL. 💯

**Review by viksit — APPROVED** [review]
just a small typo, would be great if it can be fixed

**Comment by shrihari689** [comment]
Just a note: Now Districts has "All" as option 📍 If there are any edge cases, that could be handled 📌

Everything looks good to me 🎉
Great work @minkesh 💙

File: components/Selector.jsx (hunk 2 above)
Comment (minkeshj) [reply components/Selector.jsx:17]: No the behavior will stay the same, I have just changed the function call method

**Comment by minkeshj** [comment]
@viksit @shrihari689 I have made the required changes. please review again

**Review by shrihari689 — APPROVED** [review]

**Pull request merged**